nixos/modules/programs/ente-auth.nix

```
@@ -0,0 +1,22 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.ente-auth;
+in
+{
+  options.programs.ente-auth = {
+    enable = lib.mkEnableOption "Ente Auth";
+    package = lib.mkPackageOption pkgs "ente-auth" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    services.gnome.gnome-keyring.enable = true;
```

We should only enable gnome-keyring if the user doesn't have any other keyrings like KDE Wallet enabled, because it may (in case of KDE Wallet it *does*) take priority and replace the previous keyring.

This is not the best solution, but should catch most KDE Wallet users:

```suggestion
    services.gnome.gnome-keyring.enable = lib.mkIf (
      !config.services.desktopManager.plasma6.enable
    ) true;
```

(I also tried checking whether `pkgs.kdePackages.kwallet` is added to `xdg.portalsExtraPortals` or `environment.systemPackages`, but it resulted in an infinite recursion as `services.gnome-keyring.enable` adds their package to the same lists)